gen.go

```diff
@@ -1,3 +1,3 @@
 package pbtypes
 
-//go:generate protoc --go_out=. timestamp.proto void.proto
+//go:generate protoc -I$GOPATH/src -I$GOPATH/src/github.com/gogo/protobuf/protobuf -I$GOPATH/src/github.com/gengo/grpc-gateway/third_party/googleapis -I. --gogo_out=. timestamp.proto void.proto html.proto
```

Comment:
Can you please use `gopathexec` here, so that this directive will be compatible with multiple GOPATH workspaces?

See example and details at https://github.com/sourcegraph/go-vcs/pull/67/files#diff-a0a6d3636f53a9151da2e5a2f63ec8fe.